lib/server/http/users/controller.ts

```diff
@@ -110,15 +110,15 @@ export class HTTPUsersController {
     res: Response
   ) {
     const { deactivator, redirect } = req.query;
-    const userId = (req as AuthorizedRequest<any>).user._id;
+    const userEmail = (req as AuthorizedRequest<any>).user.email;
 
     if (!deactivator || !redirect) {
       return res.status(400).send({ error: 'Missing required params' });
     }
 
     try {
       const userRequestedToBeDestroyed = await this.usersUsecase.requestUserDestroy(
-        userId, 
+        userEmail, 
```

Comment:
We should use the `user.uuid` to perform the destroy (as we will allow changing the email soon). Is there some reason to use the email instead?

Reply:
No, I used the email because that's what `requestUserDestroy` is expecting. I'll change it.